src/ocean/core/array/Search.d

```diff
@@ -958,13 +958,13 @@ unittest
 
    Performs a linear scan of haystack from [0 .. haystack.length$(RP), returning
    a count of the number of elements where pred returns true.
-  
+
```

Comment:
Not really important, but this should be in a separate commit (or not committed at all).